features/step_definitions/frontend_steps.rb

```diff
@@ -9,3 +9,19 @@
   doc = Nokogiri::HTML(html)
   assert doc.css('#services-and-information')
 end
+
+Then /^I should see an input field for entering my postcode$/ do
+  @response.body.should have_field('postcode')
+end
+
+Then /^I should see the heading "(.*?)"$/ do |title|
+  @response.body.should have_css('h1', text: title)
```

Comment:
Does this do an exact match on text?  Not sure whether the UK passport test (for example) would fail as the header only contains that text and is not an exact match.